about.html

```diff
@@ -167,7 +134,7 @@ <h3>Partner</h3> <hr>
     </ul>
 
     <div class="footer-text">
-      <img src="images/logo.svg" alt="logo">
+      <a href="#"><img src="images/logo.svg" alt="logo"></a>
       <p>© Easybank. All Rights Reserved</p>
     </div>
```

Comment:
- [ OPTIONAL ] In your about page in the mobile version but I think it would be great if you add the partner and the footer section as shown in the given design since there are in the given design and please align them as the given design as well 🔥🚀

- Your
![image](https://github.com/aradradev/easyBankTC/assets/58553711/63ef0360-e866-457c-be6b-3140ca80909a)


- Given
![image](https://github.com/aradradev/easyBankTC/assets/58553711/8dc010ee-645c-4e7d-97f9-9ff30f0c2a31)